Update log_pipeline.md (#23324)
DataDog/documentation: content/en/developers/integrations/log_pipeline.md

Validations are run automatically in your pull request. 

Three common validation errors are:
1. The `id` field in both YAML files: Ensure that the `id` field matches the `app_id` field in your integration's `manifest.json` file to connect your pipeline to your integration. 
2. Not providing the result of running the raw logs you provided against your pipeline. If the resulting output from the validation is accurate, take that output and add it to the `result` field in the YAML file containing the raw example logs.

> 3. If you send in `service` as a parameter instead of being sent in the log payload, please include the `service` field beneath your log samples within the yaml file.

Reviewer: ```suggestion
3. If you send `service` as a parameter, instead of sending it in the log payload, you must include the `service` field below your log samples within the yaml file.
```
